examples/leak_tracking/lib/main.dart

```diff
@@ -4,7 +4,7 @@ import 'package:leak_tracker/leak_tracker.dart';
 
 void main() {
   LeakTracking.start();
-  MemoryAllocations.instance.addListener(
+  FlutterMemoryAllocations.instance.addListener(
```

Comment:
It looks like this typedef comes from the package:flutter import; when first reading this I had assumed this was from package:leak_tracker. Perhaps clarify this (via comments) in this example?